SmartDeviceLink/private/SDLChoiceSetManager.m

```diff
@@ -268,6 +267,18 @@ - (void)preloadChoices:(NSArray<SDLChoiceCell *> *)choices withCompletionHandler
         __strong typeof(weakSelf) strongSelf = weakSelf;
         SDLLogD(@"Choices finished preloading");
 
+        NSMutableSet<SDLChoiceCell *> *failedChoiceUploads = [NSMutableSet set];
```

Comment:
This shouldn't happen in `preloadOp.completionBlock`, the completion block only occurs after the next operation starts running, which means if there's a preload then a present, the present will start before this code runs, which means that the present will be unaware of the failures. There needs to be another block passed to the operation which runs before completion in which code like this would run. See voice command manager for example.